applications/luci-app-tailscale/htdocs/luci-static/resources/view/tailscale/interface.js

```diff
@@ -0,0 +1,117 @@
+/* SPDX-License-Identifier: GPL-3.0-only
+ *
+ * Copyright (C) 2022 ImmortalWrt.org
+ * Copyright (C) 2024 asvow
+ */
+
+'use strict';
+'require dom';
+'require fs';
+'require poll';
+'require ui';
+'require view';
+
+return view.extend({
+	load: function() {
```

Comment:
This would look more readable as an `async` function:
```suggestion
	async load() {
```